pmlc/conversion/tile_to_pxa/tests/eltwise_logical.mlir

```diff
@@ -0,0 +1,66 @@
+// RUN: pmlc-opt -convert-tile-to-pxa -canonicalize -cse %s | FileCheck %s
+
+func @eltwise_and_f32(
+  %arg0: tensor<3x3xf32>,
+  %arg1: tensor<3x3xf32>
+) -> tensor<3x3xf32> {
+  %0 = "eltwise.logical_and"(%arg1, %arg0) : (
+    tensor<3x3xf32>,
+    tensor<3x3xf32>
+  ) -> tensor<3x3xf32>
```

Comment:
I don't think we should have the logical ops ever return anything other than `i1`

Reply:
Actually this doesn't seem legal, the definition of `EW_BinaryBoolOp` says that the result should be `EltwiseBool`. Perhaps the `EltwiseBool` predicate isn't right?